docs/quickstart/visualization/k8s-network-mapper.mdx

```diff
@@ -29,7 +29,8 @@ Before you start, you'll need a Kubernetes cluster. Having a cluster with a [CNI
 You can now install Otterize in your cluster (if it's not already installed), and optionally connect to Otterize Cloud. Connecting to Cloud lets you:
 
 1. See what's happening visually in your browser, through the "access graph";
-2. Avoid using SPIRE (which can be installed with Otterize) for issuing certificates, as Otterize Cloud provides a certificate service.
+2. Allow you to view pod public internet egress traffic.
+3. Avoid using SPIRE (which can be installed with Otterize) for issuing certificates, as Otterize Cloud provides a certificate service.
```

Comment:
We can remove this bit since it's out of date - SPIRE is no longer deployed by default

Reply:
Done.